ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/cesium/map.cesium.js

```diff
@@ -313,6 +310,27 @@ module.exports = function CesiumMap(
     onCameraMoveEnd: function(callback) {
       map.scene.camera.moveEnd.addEventListener(callback)
     },
+    doPanZoom: function(coords) {
+      const cartArray = coords.map(function(coord) {
+        return Cesium.Cartographic.fromDegrees(
+          coord[0],
+          coord[1],
+          map.camera._positionCartographic.height
+        )
+      })
+      if (cartArray.length === 1) {
+        const point = Cesium.Ellipsoid.WGS84.cartographicToCartesian(
+          cartArray[0]
+        )
+        this.panToCoordinate(point, 2.0)
+      } else {
+        const rectangle = Cesium.Rectangle.fromCartographicArray(cartArray)
+        this.panToRectangle(rectangle, {
+          duration: 2.0,
```

Comment:
Why are you using 2.0 for duration here but .5 in `panToCoordinate` function?

Reply:
`doPanZoom` is called after the gazetteer search box has a result submitted to it. The existing behavior offered a more gentle transition in this particular context and I wanted to preserve that. Otherwise it's a tad jarring.